src/manifolds/Grassmann.jl

```diff
@@ -99,10 +99,12 @@ injectivity_radius(::Grassmann, p) = π / 2
 injectivity_radius(::Grassmann, ::AbstractRetractionMethod) = π / 2
 injectivity_radius(::Grassmann, p, ::AbstractRetractionMethod) = π / 2
 
-function Base.isapprox(M::Grassmann, p, X, Y; kwargs...)
-    return isapprox(sqrt(inner(M, p, zero_vector(M, p), X - Y)), 0; kwargs...)
+function Base.isapprox(M::Grassmann, p, X, Y; atol=sqrt(max_eps(X, Y)), kwargs...)
```

Comment:
`GeneralizedGrassmann` has the same issue, it would be worth fixing as well.

Reply:
Yes, that would be great but the problem should be _exactly_ the same fix :)

Reply:
Already fixed.